scripts/sync-up-helm-chart.sh

```diff
@@ -59,17 +59,14 @@ for filename in ./config/crds/*.yaml; do
   echo "---" >> ${TEMP_CRDS_YAML}
 done
 
-# Add crd-install to make sure the CRDs can be installed first in a helm chart.
-$SED -i 's/^metadata:/metadata:\n  annotations:\n    "helm.sh\/hook": crd-install/g' "${TEMP_CRDS_YAML}"
-
 # "diff -U 4" will take 1 as return code which will cause the script failed to execute, here
 # I was force returning true to get a return code as 0.
-crd_diff=`(diff -U 4 ${TEMP_CRDS_YAML} ${CHART_FEDERATED_CRD_DIR}/crds.yaml; true;)`
-if [ -n "${crd_diff}" ]; then
-  cp -f ${TEMP_CRDS_YAML} $CHART_FEDERATED_CRD_DIR/crds.yaml
-  $SED -i '1i{{ if (or (or (not .Values.global.scope) (eq .Values.global.scope "Cluster")) (not (.Capabilities.APIVersions.Has "core.kubefed.io\/v1beta1"))) }}' ${CHART_FEDERATED_CRD_DIR}/crds.yaml
-  $SED -i '$a{{ end }}' ${CHART_FEDERATED_CRD_DIR}/crds.yaml
-fi
+#crd_diff=`(diff -U 4 ${TEMP_CRDS_YAML} ${CHART_FEDERATED_CRD_DIR}/crds/crds.yaml; true;)`
```

Comment:
we should be able to remove these commented items.

Reply:
I wasn't sure if this is still required. Now it's deleted